### PR TITLE
Update readme with new bounty information

### PR DESCRIPTION
**Pull request by layters**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -128,41 +128,67 @@ This is a one-time pay for each task. Payments will be in XMR.
...
+Libraries/Tools: OpenSSL
+Difficulty: Easy-Intermediate
+Status: Not Filled
+
+
+Details: Create an organized documentation on the neroshop codebase including functions, classes, etc.
+Bounty Reward: $50
+Libraries/Tools: Doxygen or any other tools
+Difficulty: Easy
+Status: Not Filled
```
Comment (jbakosi) [inline readme.md]: Can you take this off? I'd prefer doing it. Pretty easy and I want to do it in a way that produces a nice-looking doxygen output.

File: readme.md [hunk 2]
```diff
@@ -128,41 +128,67 @@ This is a one-time pay for each task. Payments will be in XMR.
...
+Details: Implement peer discovery system that broadcasts/multicasts messages to other peers, connects to other peers, stores their IP addresses and shares it with other neroshop nodes.
+Bounty Reward: $1000-$3000
+Files: src/daemon.cpp; src/server.cpp; src/client.cpp
+Libraries/Tools: System network libraries (Linux Sockets, Winsock2, etc.)
+Difficulty: Hard
+Status: Not Filled
+
+
+Details: Create an advance search engine using built-in LMDB database. More details will be discussed.
+Bounty Reward: $2000-$5000
```
Comment (jbakosi) [inline readme.md:140]: The bounty is okay, but I'd like to coordinate this work with whoever is up for it. If no one applies, I will do this.

**Review by jbakosi — APPROVED** [review]
Looks good, please merge.

File: readme.md (hunk 1 above)
Comment (layters) [reply readme.md]: Sure, I'll remove this specific bounty.

File: readme.md (hunk 2 above)
Comment (layters) [reply readme.md:140]: Okay, I hear you.

**Pull request merged**